src/utils/constants.py

```diff
@@ -0,0 +1,37 @@
+SYMBOLS = ['C', 'C#', 'D', 'D#', 'E',
+           'F', 'F#', 'G', 'G#', 'A', 'A#', 'B']
+REVERSE_SYMBOLS = {
+    s: i for i, s in enumerate(SYMBOLS)
+}
+REVERSE_SYMBOLS.update(
+    {
+        s.lower(): i for i, s in enumerate(SYMBOLS)
+    }
+)
+REVERSE_SYMBOLS.update(
+    {
+        'cis': 1,
+        'dis': 3,
+        'fis': 6,
+        'gis': 8,
+        'ais': 10
+    }
+)
+REVERSE_RHYTHMIC_VALUES = {
```

Comment:
I feel like `RHYTHMIC_VALUES` is misleading. For me it means that the dictionary will return a rhytmic value. Dicts should be generally named VALUES, VALUES_BY_KEY or KEY_TO_VALUE

Reply:
Fixed